Week-13/weather/index.js

```diff
@@ -0,0 +1,14 @@
+const express = require("express");
```

Comment:
Do not commit node_modules. Add the entry in .gitignore so that they are ignored

Reply:
Changes done